ParseEmailFiles - fixed an issue where 'from' header has '\r\n' (#9728)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_2_69.md

#### Scripts
##### ParseEmailFiles

> - Fixed an issue where an email has 'From' header that contains '\r\n' and caused from address to be invalid.

Reviewer: ```suggestion
- Fixed an issue where the sending address was invalid in case the email's 'From' header contained '\r\n'.
```
